x-pack/legacy/plugins/monitoring/common/format_timestamp_to_duration.js

```diff
@@ -48,7 +48,7 @@ export function formatTimestampToDuration(timestamp, calculationFlag, initialTim
   }
 
   return duration
-    .replace(/ 0 mins$/, '')
-    .replace(/ 0 hrs$/, '')
-    .replace(/ 0 days$/, ''); // See https://github.com/jsmreese/moment-duration-format/issues/64
+    .replace(/ -?0 mins$/, '')
```

Comment:
This file also lives in `x-pack/plugins/monitoring/common`

Reply:
Goot catch 👍 Fixed